data-prepper-plugins/sqs-source/src/main/java/org/opensearch/dataprepper/plugins/source/sqs/BulkSqsMessageHandler.java

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
Please use the longer copyright header.

https://github.com/opensearch-project/data-prepper/blob/90575b1de56f82f44d1af36f31ff4b077a627bd7/CONTRIBUTING.md#license-headers

```
/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 *
 * The OpenSearch Contributors require contributions made to
 * this file be licensed under the Apache-2.0 license or a
 * compatible open source license.
 *
*/
```

Reply:
added to every file